glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+## Application
+
+A process tree launched inside a [container](#container).
+
+## Bundle
+
+A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
```

Comment:
On Tue, Aug 11, 2015 at 09:26:58PM -0700, Brandon Philips wrote:
> > +## Bundle
> > +
> > +A [directory structure](bundle.md) that is written ahead of time, distributed, and used to seed the runtime for creating [containers](#container) and launching [applications](#application).
> 
> launching an application? A bundle only specifies launches a single
> application process.

One application (which can result in a multi-process tree) per runtime
invocation yes.  But you can use the same bundle to launch several
copies of the application.

And you can also have several config choices in a single bundle.  My
proposed docs for this are in this subsection [1] of #76.  That PR was
rejected for other reasons, but that particular subsection still
applies to the current spec and at least the runC implementation.

[1]: https://github.com/wking/opencontainer-specs/blob/13349ba471ffb5a287527bf585fdb9cbcc0d7b95/bundle.md#alternative-configurations